tests/jest/setup/libraries.js

```diff
@@ -0,0 +1,21 @@
+import '@testing-library/jest-dom/extend-expect';
+
+global.PropTypes = require( 'prop-types' );
+global.React = require( 'react' );
+global.ReactDOM = require( 'react-dom' );
+global.react = global.React;
+global[ 'react-dom' ] = global.ReactDOM;
+global.__ = ( text ) => text;
+
+// Mock jQuery
+const jQuery = new class {
+	constructor() {
+		return new Proxy( this, {
+			get: () => {
+				return jest.fn();
+			},
+		} );
```

Comment:
Why Proxy?

Reply:
The alternative is to create a mock for each jQuery function I use. Proxy, in this matter, acts like `__call` magic method, and catches every jQuery function invocation.

Reply:
You have method in jest to mock the entire module